### DISCUSSION TITLE
graphwiz dot file incorrectly rendered

### DISCUSSION THREAD

**Discussion by gverzino63** [tamlok/vnote#2441] [Q&A]
Hi all,
I'm not able to render a graphwiz dot file in vnote.
The following example found in internet is wrongly displayed.

Anyone knows how to configure vnote to render it correctly?

`
```dot

digraph H {

  parent [
   shape=plaintext
   label=<
     <table border='1' cellborder='1'>
       <tr><td colspan="3">The foo, the bar and the baz</td></tr>
       <tr><td port='port_one'>First port</td><td port='port_two'>Second port</td><td port='port_three'>Third port</td></tr>
     </table>
  >];

  child_one [
   shape=plaintext
   label=<
     <table border='1' cellborder='0'>
       <tr><td>1</td></tr>
     </table>
  >];

  child_two [
   shape=plaintext
   label=<
     <table border='1' cellborder='0'>
       <tr><td>2</td></tr>
     </table>
  >];

  child_three [
   shape=plaintext
   label=<
     <table border='1' cellborder='0'>
       <tr><td>3</td></tr>
     </table>
  >];

  parent:port_one   -> child_one;
  parent:port_two   -> child_two;
  parent:port_three -> child_three;

}
`

**Comment by tamlok** [comment]
![image](https://github.com/vnotex/vnote/assets/5763157/47350fb0-e4d5-4c58-9ca9-f01640902abd)

**Reply by gverzino63** [reply]
Hi tamlok,

as you can see at https://renenyffenegger.ch/notes/tools/Graphviz/examples/index the render should be the following one.

Do you have any idea why vnote render it incorrectly?

Does vnote uses an internal version of dot?

![image](https://github.com/vnotex/vnote/assets/64150088/93fd9795-abad-4c19-82d7-c3af24ca68f1)

**Reply by tamlok** [reply]
VNote uses a web-based dot by default. You can specify local dot executable in Settings.
